src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubConfiguration.cs

```diff
@@ -69,6 +69,11 @@ public EventHubConfiguration()
         }
 
         /// <summary>
+        /// Sets the frequency at which to checkpoint event batches.
+        /// </summary>
+        public int BatchesPerCheckpoint { get; set; } = 1;
```

Comment:
Naming is totally up in the air, I struggled to come up with something I liked :)